v2.1/sql-name-resolution.md

```diff
@@ -91,6 +91,15 @@ The current schema is used as target schema when creating a new object if the na
 
 - You can inspect the current schema using the special built-in function/identifier `current_schema()`.
 
+## Index name resolution
+
+CockroachDB supports the following ways to specify an index name for statements that require one (e.g., [`DROP INDEX`](drop-index.html), [`ALTER INDEX ... RENAME`](alter-index.html), etc.):
+
+1. Index names are resolved relative to a table name using the `@` character, e.g. `DROP INDEX tbl@idx;`.  This is the default and most common syntax.
+2. Index names are resolved by searching all tables in the current schema to find a table with an index named `idx`, e.g., `DROP INDEX public."idx";`  This syntax is necessary for Postgres compatibility because Postgres index names live in the schema namespace such that e.g. `public."idx"` will resolve to the index `idx` of some table in the public schema.  This capability is used by some ORMs.
```

Comment:
say "e.g, `DROP INDEX idx;` or `DROP index public.idx;`

1) The double quotes are not needed I think?

2) the schema prefix is optional.

Reply:
Fixed by making the following changes: removed quotes around index names; updated examples as suggested; and noted that the prefix is optional.

Thanks @knz !